RecoMET/METProducers/src/BeamHaloSummaryProducer.cc

```diff
@@ -258,7 +258,7 @@ void BeamHaloSummaryProducer::produce(Event& iEvent, const EventSetup& iSetup)
     {
       if( iWedge->NumberOfConstituents() > T_EcalPhiWedgeConstituents )
         GlobalTightId = true;
-      if( std::abs(iWedge->ZDirectionConfidence() > T_EcalPhiWedgeConfidence) )
+      if( std::abs(iWedge->ZDirectionConfidence()) > T_EcalPhiWedgeConfidence )
```

Comment:
@slava77 
Apparently this bug is at least 6 years old, thanks for spotting it.  
This can possibly affect one of the filters stored in BeamHaloSummary. This filter is however not used by anyone at the moment.